### PR TITLE
Fixed the component tower locking up when fired upon during the make animation.

### LINKED ISSUE #11349: [TS] GDI Component Tower bug.
When GDI component tower is placed and being fired upon during Make Animation, it skips the make animation and locks up. You can't sell it, you can't have a turret on it, in other words it becomes a useless drain of power until destroyed.
I noticed it's related to WithWallSpriteBody: trait and as long as it's present the problem persists.

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/11349.

### CODE REVIEW

**Review by pchote — APPROVED** [review]
Fix confirmed, thanks!

This can be more easily tested by adding `Tick: 400` to `gactwr`'s `make` sequence to slow it down, and then using a single mammoth mk 2 shot to damage it while the animation runs.

File: mods/ts/sequences/structures.yaml [hunk 1]
```diff
@@ -1192,6 +1192,8 @@ gactwr:
 		Start: 2
 		ShadowStart: 5
 		Tick: 400
+	invisible: null
```
Comment (pchote) [inline mods/ts/sequences/structures.yaml:1195]: TIL that TS comes with a `null.shp`.

Comment (Mailaender) [reply mods/ts/sequences/structures.yaml:1195]: Programmer artwork.

**Review by Smittytron — APPROVED** [review]
Works

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**